source/simulator/core.cc

```diff
@@ -1591,6 +1591,16 @@ namespace aspect
 
     if (parameters.resume_computation == false)
       {
+        // See if we want to skip the initial conditions setup during the pre-
+        // refinement phase. If set true, the solvers won't be executed. An
```

Comment:
If set true, the solvers won't be executed -> If set **to** true, the solver **will not** be executed/